lib/theme.js

```diff
@@ -1,91 +1,106 @@
 'use strict';
-import bower from 'bower';
 import when from 'when';
 import Path from 'path';
-import mkdirp from 'mkdirp';
 import _ from 'lodash';
+import fs from 'fs';
 import File from 'fs-extra';
-import del from 'del';
 import elegantSpinner from 'elegant-spinner';
 import logUpdate from 'log-update';
-import Dir from './dir';
 import 'source-map-support/register';
 
 let frame = elegantSpinner();
 
-
 /**
- * The class that installs themes.
+ * The class that locates themes
  * @class  Theme
  */
 class Theme {
   constructor(options) {
-    // Set hidden path
-    let hiddenPath = Path.join(Dir.getHomeDir(), '.doxx');
 
-    // Set template options
+    let resolvedTheme = this.locateTheme(options.theme);
+
     this.options = {
       theme: {
-        name: options.theme,
-        path: hiddenPath
-      },
-      template: {
-        path: options.template.path
+        name: resolvedTheme.theme,
+        path: resolvedTheme.path
       },
       target: {
         path: options.target
       }
     };
+
   }
 
-  /** 
-   * Installs the theme and its assets statically
-   * @param {Object} options The options
-   * @jsfiddle https://jsfiddle.net/iwatakeshi/k5xsvoez/embedded/
-   * @return {Function} The promise
+  /**
+   * Find the theme specified
    */
-  static staticInstall(options) {
-      let final = when.defer();
-      /*
-        Required options:
-        
-        theme.name    (the name of the theme)
-        theme.path    (the path to store the theme)
-        target.path   (the path to store the compiled content)
-        template.path (the template path) *for test purposes only*
-       */
+  locateTheme(theme) {
+
+    const DEFAULT_THEME = 'doxx-theme-default';
+    const doxxBasePath = Path.resolve(__dirname, '..');
```

Comment:
@cliftonc If you can combine theses const, that would be great as well. Something like:
```javascript
var paths = {
  doxx: { base: {/*...*/} },
}
_.extend((paths.target = {}),{ /*...use the base...*/});



```
This just makes it easier to read.

If you would like to make this immutable, then we can use Facebook's [immutable].(https://facebook.github.io/immutable-js/)